README.md

```diff
@@ -8,7 +8,11 @@
 
 ## Getting Started
 
-To build it, we use [mill](http://www.lihaoyi.com/mill/). I you want to run the specification tests, just type:
+To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not available on on Unix boxes, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
```

Comment:
```suggestion
To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not packaged all Linux distributions, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
```

Technically, it is available on some Unixes, e.g. MacOS through brew and Arch Linux in AUR, and FreeBSD has a port. Main lack is for Debian based and Redhat based distros :)